clip.py

```diff
@@ -123,6 +140,27 @@ def channels(self, clip):
         else:
             return self.data[clip.audio_file].shape[1]
 
+    @property
+    def outputs(self):
+        seen = set()
```

Comment:
At some point I needed a consistent order and set doesn't guarantee that, but I'm not sure anymore if the order still matters now.
But I'm putting back a the song.outputs list, so outputs can exist independently of whether they exist anyways next.